{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "8f3ae3ff_823f7359",
        "filename": "container-images/gerrit-init/tools/gerrit-initializer/initializer/tasks/init.py",
        "patchSetId": 2
      },
      "lineNbr": 138,
      "author": {
        "id": 1065256
      },
      "writtenOn": "2023-05-17T13:55:44Z",
      "side": 1,
      "message": "If I remove an entry for `.Values.gerrit.additionalConfigMaps` the symlink is not deleted, so something still doesn\u0027t work. Could you give it another check? Thanks!",
      "range": {
        "startLine": 138,
        "startChar": 20,
        "endLine": 138,
        "endChar": 39
      },
      "revId": "df72d9eecd118727f501c996dea7651eccb2145a",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "108c0255_7dcbc952",
        "filename": "container-images/gerrit-init/tools/gerrit-initializer/initializer/tasks/init.py",
        "patchSetId": 2
      },
      "lineNbr": 138,
      "author": {
        "id": 1086278
      },
      "writtenOn": "2023-05-19T03:33:08Z",
      "side": 1,
      "message": "Does this happen for replicas? I checked and saw this is skipped for initialized replicas. I\u0027ve upload a new PS. PTAL.",
      "parentUuid": "8f3ae3ff_823f7359",
      "range": {
        "startLine": 138,
        "startChar": 20,
        "endLine": 138,
        "endChar": 39
      },
      "revId": "df72d9eecd118727f501c996dea7651eccb2145a",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "06b1828d_14ecab98",
        "filename": "container-images/gerrit-init/tools/gerrit-initializer/initializer/tasks/init.py",
        "patchSetId": 2
      },
      "lineNbr": 138,
      "author": {
        "id": 1065256
      },
      "writtenOn": "2023-06-12T13:38:43Z",
      "side": 1,
      "message": "I tested it on a primary Gerrit. I added an additional ConfigMap for the data directory, installed the helm chart and then removed the ConfigMap and updated the chart. Afterwards, the symlink still existed.",
      "parentUuid": "108c0255_7dcbc952",
      "range": {
        "startLine": 138,
        "startChar": 20,
        "endLine": 138,
        "endChar": 39
      },
      "revId": "df72d9eecd118727f501c996dea7651eccb2145a",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "7af5e392_0707625d",
        "filename": "container-images/gerrit-init/tools/gerrit-initializer/initializer/tasks/init.py",
        "patchSetId": 2
      },
      "lineNbr": 138,
      "author": {
        "id": 1086278
      },
      "writtenOn": "2023-07-09T04:24:20Z",
      "side": 1,
      "message": "I\u0027ve upload a new PS. I also see this happens for configuration, i.e. files under etc/. Should this get fixed too?",
      "parentUuid": "06b1828d_14ecab98",
      "range": {
        "startLine": 138,
        "startChar": 20,
        "endLine": 138,
        "endChar": 39
      },
      "revId": "df72d9eecd118727f501c996dea7651eccb2145a",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}